js/views/modals/purchase/ConfirmWallet.js

```diff
@@ -0,0 +1,68 @@
+import $ from 'jquery';
+import app from '../../../app';
+import loadTemplate from '../../../utils/loadTemplate';
+import BaseVw from '../../baseVw';
+import { integerToDecimal } from '../../../utils/currency';
+
+export default class extends BaseVw {
+  constructor(options = {}) {
+    if (!options.displayCurrency) {
+      throw new Error('Please provide a display currency.');
+    }
+
+    super(options);
+    this.options = options;
+    this.fee = false;
+
+    this.confirmedAmount = app.walletBalance.get('confirmed');
```

Comment:
Why keep track locally of the confirmedAmount instead of getting it from the walletBalance model as needed? You end up having a local variable that you have to keep in sync with that model.